.github/workflows/test_singularity.yml

```diff
@@ -2,7 +2,7 @@ name: "Linting and Testing (Singularity)"
 
 on:
   push:
-    branches: [temsim-test-in-container]
+    branches: [simspi-test-singularity]
```

Comment:
should this be called test-container? (I thought we removed singularity?) we also do not need `simspi`, since it is on the simSPI repo.

Reply:
ah good point! 

I kept the test_singularity workflow but it's only triggered if we push on this branch.

So maybe in terms of renaming:
- `test_singularity.yml` (keep name): rename trigger branch to `test-singularity`
- `test_docker.yml`: I will split it in a way similar to what you did on ioSPI once we merge this branch (linting decoupled from testing). The trigger branch will be rename `test-container`.